src/Ens/JobeetBundle/DataFixtures/ORM/LoadCategoryData.php

```diff
@@ -0,0 +1,42 @@
+// src/Ens/JobeetBundle/DataFixtures/ORM/LoadCategoryData.php
```

Comment:
@PhotonSapiens 
You missed `<?php` at the beginning - so file is interpreted as plain text, not code